src/migrations/versions/809dd14b4b64_bd_enums_to_varchar.py

```diff
@@ -0,0 +1,75 @@
+"""bd enums to varchar
```

Comment:
Add the downgrade, also remove the enum types definition from postgres not just the columns

Reply:
Done.